### PR TITLE
Add open/close user in Activity

### PR DESCRIPTION
**Pull request by Toldy**
Looks like that:
image

### CODE REVIEW

File: src/web/components/ActivityComponent.html [hunk 1]
```diff
@@ -53,7 +63,29 @@
...
+                    return false
+                }
+            }
+        },
+        methods: {
+            editIsClosed: function (user, value) {
+                var cookie = {}
+                try { cookie = JSON.parse(document.cookie) } catch (e) {}
+                cookie[user.name] = value
+                document.cookie = JSON.stringify(cookie)
```
Comment (Dewep) [inline src/web/components/ActivityComponent.html]: It's very weird to do that (cookies are not stored as a JSON string). I don't think that works on all browsers.

**Review by Dewep — CHANGES_REQUESTED** [review]
You should take a look on localStorages instead of using cookies. You can find an example here: [zajdaf/Transmi:/src/web/public/lib/API.js#L2-L26](https://github.com/zajdaf/Transmi/blob/19c8301555a2e278c713467ecf8ebd4cd349af07/src/web/public/lib/API.js#L2-L26).

**Comment by Toldy** [comment]
Ofc you were right, my cookie trick didn't work for anything else Chrome 👎

**Review by Dewep — APPROVED** [review]
Thank you for the modifications. This is not important for this project, but you should think to check that `localStorage` is defined (not the case for the old browsers: [caniuse.com/#search=localStorage](http://caniuse.com/#search=localStorage)) (or do a simple `try catch`).

**Pull request merged**